### DISCUSSION TITLE
Restore to development from live, option to clean content/media

### DISCUSSION THREAD

**Discussion by alexanderkarlsson** [umbraco/Umbraco.Cloud.Issues#1009] [Ideas]
When you do a restore from the live to development workspace it merges the data. It would be a nice feature to have an option to restore with a clean media and content option.

**Comment by nul800sebastiaan** [comment]
Thanks @alexanderkarlsson - we understand the request and it's an interesting idea. Just so you know, for now you are able to do this by deleting the development environment and adding a new one. 

We've put this on the list of ideas to make things easier in the future though! 👍

**Comment by SimonAndrews** [comment]
Having recently started developing on the Umbraco cloud I've found this would be a useful feature to have. I've had to reset the environment manually a few times now, being able to do this with a few button clicks would definitely be a good feature.

**Comment by azure-devops-sync** [comment]
AB#675
